### DISCUSSION TITLE
明细表当我自定义dataCell后，无法选中整行，是要加上什么吗？求大神指导一下

### DISCUSSION THREAD

**Discussion by Chasen-Zhang** [antvis/S2#1756] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/15354455/190374634-e9dccfaa-f580-4409-ac95-e02fbf300879.png)
当我自定义dataCell后，无法选中整行，只能选中序号，是要加上什么吗？求大神指导一下
![image](https://user-images.githubusercontent.com/15354455/190374229-553ec17e-3789-4e26-aa22-334dd5f6a85b.png)

**Comment by Chasen-Zhang** [comment]
明细表会出这个问题

**Comment by YardWill** [comment, marked as answer]
序列行是单独处理的，所以这里第一列的自定义Cell 需要extends TableSeriesCell，其他自定义DataCell。
